src/config.rs

```diff
@@ -2980,9 +2979,30 @@ impl TiKvConfig {
             );
         }
 
+        #[cfg(not(any(test, feature = "textexport")))]
+        self.raft_log_gc_limit_adjust();
+
         Ok(())
     }
 
+    pub fn raft_log_gc_limit_adjust(&mut self) {
+        // Raft log gc limit must be 75% region size at least;
+        if self.raft_store.raft_log_gc_size_limit < self.coprocessor.region_split_size * 3 / 4 {
```

Comment:
How about also using `Option`? This should not be a hard limit.

Reply:
Too many changes were introduced previously, will try again if it was needed.

Reply:
User should be allowed to use a smaller value for aggressive disk space reclaim.